ivadomed/losses.py

```diff
@@ -233,3 +234,60 @@ def forward(self, input, target):
 
         # TODO: Take the opposite?
         return focal_tversky_sum / n_classes
+
+
+class L2loss(nn.Module):
```

Comment:
don't forget to update the documentation that lists the losses: https://github.com/neuropoly/ivado-medical-imaging/blob/master/docs/source/api_ref.rst#modivadomedlosses

Reply:
docstrings needed

Reply:
It is just below (as there is no parameter, it is just a description)

Reply:
done in: cf549d9 and corrected in ceb746d